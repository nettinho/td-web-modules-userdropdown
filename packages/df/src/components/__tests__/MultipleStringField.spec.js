import React from "react";
import { shallowWithIntl } from "intl-stub";
import { MultipleStringField } from "../MultipleStringField";

describe("<MultipleStringField />", () => {
  const label = "label";
  const name = "name";
  const required = true;
  const value = [];
  const onChange = jest.fn();
  const props = { label, name, required, value, onChange };

  it("matches the latest snapshot", () => {
    const wrapper = shallowWithIntl(<MultipleStringField {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
