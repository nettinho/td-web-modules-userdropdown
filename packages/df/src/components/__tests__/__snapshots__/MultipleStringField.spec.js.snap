// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MultipleStringField /> matches the latest snapshot 1`] = `
<FormField
  required={true}
>
  <label>
    label
  </label>
  <FormDropdown
    allowAdditions={true}
    as={[Function]}
    control={[Function]}
    fluid={true}
    multiple={true}
    name="name"
    onChange={[MockFunction]}
    options={
      Array [
        Object {
          "key": 0,
          "text": "",
          "value": "",
        },
      ]
    }
    placeholder="fields.dropdown.placeholder"
    search={true}
    selection={true}
    value={Array []}
  />
</FormField>
`;
